source/routing/specifying-a-routes-model.md

```diff
@@ -156,61 +75,11 @@ export default Ember.Route.extend({
 In the `model` hook for routes with dynamic segments, it's your job to
 turn the ID (something like `47` or `post-slug`) into a model that can
 be rendered by the route's template. In the above example, we use the
-photo's ID (`params.photo_id`) to construct a URL for the JSON
-representation of that photo. Once we have the URL, we use jQuery to
-return a promise for the JSON model data.
+photo's ID (`params.photo_id`) as an argument to Ember Data's `findRecord`
+method.
 
 Note: A route with a dynamic segment will only have its `model` hook called
 when it is entered via the URL. If the route is entered through a transition
 (e.g. when using the [link-to](../../templates/links) Handlebars helper), then a model context is
 already provided and the hook is not executed. Routes without dynamic segments
 will always execute the model hook.
-
-### Refreshing your model
```

Comment:
My intuition says too far afield - my informal gut check is if it's not something I think will benefit more than 20% of users, it shouldn't go in the Guides. @locks what do you think?